commit b434205f3ea324e21f04017c12650b6fe02bccce
Author: Phat Huynh <sleepyorion@gmail.com>
Date:   Sun Jul 2 05:54:45 2017 -0700

    Fixed a use case where attribute 'name' in metadata.rb resulted in cookbook
    folder being created in uppercase.

diff --git a/lib/kitchen/provisioner/chef/common_sandbox.rb b/lib/kitchen/provisioner/chef/common_sandbox.rb
index a9ce229..4fc05f3 100644
--- a/lib/kitchen/provisioner/chef/common_sandbox.rb
+++ b/lib/kitchen/provisioner/chef/common_sandbox.rb
@@ -150,6 +150,9 @@ module Kitchen
                                                                         "The metadata.rb does not define the 'name' key." \
                                                                           " Please add: `name '<cookbook_name>'` to metadata.rb and retry")
 
+          # convert the cookbook name from uppercase to lowercase
+          cb_name.downcase!
+
           cb_path = File.join(tmpbooks_dir, cb_name)
 
           glob = Dir.glob("#{config[:kitchen_root]}/**")
